### PR TITLE
[Logic] `az logic integration-account`: Add new command group `map` to support manage integration account map

### PR DESCRIPTION
**Pull request by Jing-song**
---

* Add new command group `az logic integration-account map` to support manage integration account map
* Upgrade SDK to fix deserialization errors
AAZ model https://github.com/Azure/aaz/pull/219
close https://github.com/Azure/azure-cli/issues/25687
close https://github.com/Azure/azure-cli-extensions/issues/6304

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Logic

File: src/logic/azext_logic/manual/custom.py [hunk 1]
```diff
@@ -35,3 +69,114 @@ def logic_workflow_update(cmd,
...
+class MapCreate(_MapCreate):
+    @classmethod
+    def _build_arguments_schema(cls, *args, **kwargs):
+        from azure.cli.core.aaz import AAZFileArg
+        args_schema = super()._build_arguments_schema(*args, **kwargs)
+        args_schema.map_content = AAZFileArg(
+            options=["--map-content"],
+            help="The content.",
+        )
+        args_schema.content._registered = False
```
Comment (kairu-ms) [inline src/logic/azext_logic/manual/custom.py:155]: The conent argument by default can support file input like json and yaml files, so it doesn't to add another argument.

File: src/logic/azext_logic/manual/custom.py [hunk 2]
```diff
@@ -3,7 +3,43 @@
...
+        'access_control': definition.get(
+            'accessControl', access_control),
+        'integration_account': integration_account,
+        'integration_service_environment': integration_service_environment,
+        'definition': definition['definition'],
+        'parameters': definition.get('parameters', None)
+    }
+    return client.create_or_update(resource_group_name=resource_group_name,
+                                   workflow_name=name,
+                                   workflow=workflow)
```
Comment (kairu-ms) [inline src/logic/azext_logic/manual/custom.py:42]: What's these function used for?

Comment (Jing-song) [reply src/logic/azext_logic/manual/custom.py:42]: After updating the SDK, the `create_or_update` parameter was changed, `az logic workflow create` use it

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**